projects/storefrontlib/src/lib/cms-lib/search-box/search-box.component.html

```diff
@@ -21,19 +22,9 @@
       (click)="submitSearch()"
       [disabled]="!searchBoxControl?.value"
     >
-      <svg
-        class="cx-icon "
-        xmlns="http://www.w3.org/2000/svg "
-        viewBox="-4472 4760 26 26 "
-      >
-        <path
-          id="Tracé_982 "
-          data-name="Tracé 982 "
-          d="M9.75,19.5a9.241,9.241,0,0,0,6.067-2.167l8.342,8.342a1.072,1.072,0,0,0,1.517-1.517l-8.342-8.342A9.854,9.854,0,0,0,19.5,9.75,9.75,9.75,0,1,0,9.75,19.5Zm0-17.333A7.583,7.583,0,1,1,2.167,9.75,7.537,7.537,0,0,1,9.75,2.167Z "
-          transform="translate(-4472 4760) "
-        />
-      </svg>
+      <cx-icon [type]="iconTypes.SEARCH"></cx-icon>
     </button>
+
     <!-- searchbox button mobile -->
     <button
```

Comment:
So you just replaced the desktop and tablet icon. You should also replace the one in the button for mobile